crates/completion/src/completions/keyword.rs

```diff
@@ -88,6 +88,7 @@ pub(crate) fn complete_expr_keyword(acc: &mut Completions, ctx: &CompletionConte
         add_keyword(ctx, acc, "loop", "loop {$0}");
         add_keyword(ctx, acc, "if", "if $0 {}");
         add_keyword(ctx, acc, "if let", "if let $1 = $0 {}");
+        add_keyword(ctx, acc, "for", "for $0 in $1 {}");
```

Comment:
```suggestion
        add_keyword(ctx, acc, "for", "for $1 in $0 {}");
```
We want the order swapped here, `$0` is always the last cursor position

Reply:
I've changed it, thank you for the suggestion 🙂!